rest_framework_latex/templatetags/rest_framework_latex.py

```diff
@@ -11,3 +18,14 @@ def latex_safe(value):
     # http://www.cespedes.org/blog/85/how-to-escape-latex-special-characters
     value = escape_latex(value)
     return mark_safe(value)
+
+
+@register.filter(name='latex_markdown', is_safe=True)
+def latex_markdown(value):
+    if not has_markdown:
+        raise RuntimeError('CommonMark needs to be installed')
```

Comment:
Is a runtime error the best choice here or should we use `ImproperlyConfiguredError`?